whois-nrtm/src/main/java/net/ripe/db/whois/nrtm/BaseNrtmServerChannelInitializer.java

```diff
@@ -0,0 +1,79 @@
+package net.ripe.db.whois.nrtm;
+
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.Unpooled;
+import io.netty.channel.Channel;
+import io.netty.channel.ChannelInitializer;
+import io.netty.channel.ChannelPipeline;
+import io.netty.handler.codec.DelimiterBasedFrameDecoder;
+import io.netty.handler.codec.string.StringDecoder;
+import io.netty.handler.codec.string.StringEncoder;
+import net.ripe.db.whois.common.pipeline.MaintenanceHandler;
+
+import java.nio.charset.StandardCharsets;
+
+
+abstract class BaseNrtmServerChannelInitializer extends ChannelInitializer<Channel> {
+
+    private static final ByteBuf LINE_DELIMITER = Unpooled.wrappedBuffer(new byte[]{'\n'});
+
+    private static final long MEMORY_SIZE_UNLIMITED = 0;
+    private static final long TIMEOUT_SECONDS = 60L;
+    private static final int POOL_SIZE = 32;
+
+    private final StringDecoder stringDecoder = new StringDecoder(StandardCharsets.UTF_8);
+    private final StringEncoder stringEncoder = new StringEncoder(StandardCharsets.UTF_8);
+
+//    TODO [DA] ensure similar properties are configured for Netty4
```

Comment:
Use DefaultEventExecutor?

https://netty.io/wiki/new-and-noteworthy-in-4.0.html#no-more-executionhandler---its-in-the-core

Reply:
Indeed. I also noticed `DefaultEventExecutor` but also saw it has less configuration options than the execution handler it is replacing. My intention is to revisit it and ensure it has (or can be configured to have) the same characteristics as what we have before.

Reply:
👍